src/gui/layertree/qgslayertreeviewdefaultactions.cpp

```diff
@@ -283,12 +283,10 @@ void QgsLayerTreeViewDefaultActions::showFeatureCount()
 
 void QgsLayerTreeViewDefaultActions::zoomToLayer( QgsMapCanvas *canvas )
```

Comment:
Given the current name in the API, I think we'll need a new `zoomToLayers` method (and the current one can be deprecated).

Reply:
You mean like this commit? 

https://github.com/TurboGraphxBeige/QGIS/commit/4cb616df1077d988b8dec0be8710042bf5971339

Reply:
For the sake of my new contributor's mind and soul...
You mean like this commit?

https://github.com/TurboGraphxBeige/QGIS/commit/e5ae5c47c91e2be90d0e48268a2b7c776e870853

Reply:
@TurboGraphxBeige yep, looks perfect

Reply:
Alright. Is it better if I move this commit into this PR or should I make a second PR?

Reply:
Just cherry pick it into this branch please !